internal/app/backend/backend_service.go

```diff
@@ -50,6 +61,32 @@ func newBackend(cfg config.View) (*backendService, error) {
 		cfg: cfg,
 	}
 
+	if cfg.GetBool("tls.enabled") {
```

Comment:
Move this code to rpc/ with signature like. `func CreateHTTPClient(cfg config.View, address string) (http.Client, error)` and `func CreateGRPCClient(cfg config.View, address string) (grpc.ClientConn, error)`.

That will prevent the need from exposing the TLS utils.